terraform_cft/00_projects/variables.tf

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 Google LLC. This software is provided as-is, without warranty
+ * or representation for any use or purpose. Your use of it is subject to your 
+ * agreement with Google.  
+ */
+variable "gcp_billing_id" {
+  description = "The billing id to use with projects."
+}
+
+variable "gcp_org_id" {
+  description = "The ID of the GCP organization where the projects will be created"
+}
+
+variable "folder_id" {
+  description = "The ID of the folder where the projects will be created"
+}
+
+variable "shared_services_project" {
+  description = "The ID of the shared services project"
+}
+
+variable "application_project" {
+  description = "The ID of the sample application project"
+}
+
+variable "project_services" {
```

Comment:
Could a default be provided?

Reply:
Cannot provide default value to this one since it references the project ID, which will be different for each person deploying this.

Reply:
My suggestion would be to combine a default with the `random_project_id` variable for project factory which will add a random suffix to enable uniqueness.

Reply:
I was already using the random suffix. So I've added default project IDs to simplify initial setup.